articles/storage/common/azure-defender-storage-configure.md

```diff
@@ -25,11 +25,11 @@ Analyzed transactions of Azure Blob Storage include operation types such as `G
 **Defender for Storage doesn't access the Storage account data, doesn't require you to enable access logs, and has no impact on Storage performance.**
 
 > [!NOTE]
-> A new pricing is now available for Microsoft Defender for Cloud that charges you according to the number of storage accounts that you protect (per-storage account).
+> Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, and high-volume transactions may incur additional overage charges. This new pricing plan will also include all new security features and detections.
```

Comment:
```suggestion
> Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, where high-volume transactions may incur additional overage charges. This new pricing plan includes all new security features and detections.
```

Reply:
@bmansheim - "This new pricing plan **will** also include all new security features and detections." - this intentionally "will" because today there are no new features and detections, but there will be in the future, we wanted to emphasize that customers should move to the new plan because the good stuff will only go to that plan (in the future)

Reply:
Updated my suggestion

Reply:
@bmansheim - regarding "Customers with the per-storage account pricing plan will receive new security features and detections as they are released.". The emphasize is to nudge customers to move to the new pricing plan and not describe that customers on that plan will receive new features. We want them to understand that new features will **only** be there, and they should move to that plan